src/redis4net/Layout/LogMessageLayout.cs

```diff
@@ -207,7 +210,22 @@ private string GetValueFromPattern(LoggingEvent loggingEvent, string pattern)
 			}
 		}
 
-		private static long GetSyslogSeverity(Level level)
+        private static int GetLevelAsNumber(Level level)
+        {
+            var levels = new Dictionary<string, int>
```

Comment:
Does it matter that this is created on each run? Couldn't it just as well be a constant? Maybe it's an OK thing in the C# world?